### PR TITLE
LPD-39338 Apply scroll to the list of fields in form entries

### PR DESCRIPTION
**Pull request by igor-franca**
Related [ticket](https://liferay.atlassian.net/browse/LPD-39338)

### CODE REVIEW

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:sf

File: modules/test/playwright/tests/dynamic-data-mapping-form-web/formEntries.spec.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		await formBuilderSidePanelPage.addFieldByDoubleClick('Text');
+
+		await formBuilderSidePanelPage.label.fill(`Text ${index}`);
+
+		await formBuilderSidePanelPage.clickBackButton();
+	}
+
+	await formBuilderPage.clickSaveButton();
+
+	await formBuilderPage.publishButton.click();
```
Comment (larissacribeiro) [inline modules/test/playwright/tests/dynamic-data-mapping-form-web/formEntries.spec.ts:43]: hey @igor-franca, testing this locally, I think we don't need those two steps, considering that publishing will automatically save it, and that inside openFormSubmissionMethod() method , we are already publishing the forms as well. So this alone should cover the steps We need to achieve the form submission page, what do you think? You can test it as well and let me know!

**Review by larissacribeiro — CHANGES_REQUESTED** [review]
Hey @igor-franca, it looks good to me! I discovered the reason that it was failling to me locally, I had to enable this property on my portal-ext.properties: `com.liferay.portal.kernel.upload.FileItem.threshold.size=2097152` to allow the form content size to be larger than 1 MB, in this case to 2MB, otherwise it throws an error. 
Can you check if you also have this property enabled? I think We might have to add it to our directory properties.

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:playwright

**Review by larissacribeiro — APPROVED** [review]
LGTM!! 🎉

**Comment by igor-franca** [comment]
ci:forward

**Comment by igor-franca** [comment]
ci:forward

**Comment by igor-franca** [comment]
ci:report:24898

**Comment by igor-franca** [comment]
ci:report:44338004

**Comment by igor-franca** [comment]
ci:forward:force

**Comment by igor-franca** [comment]
ci:forward:force

**Comment by igor-franca** [comment]
Resent [here](https://github.com/brianchandotcom/liferay-portal/pull/155260)

**Pull request closed without merging**